.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "data"]
 	path = data
-	url = https://github.com/Bot-Devel/Quote-Finder-Data.git
```

Comment:
Edit the name back to the original url i.e. https://github.com/Bot-Devel/Quote-Finder-Data.git